### PR TITLE
[#960][FOLLOWUP] fix(dashboard): Fix get_pid_file_name function for the dashboard.

### PR DESCRIPTION
**Pull request by rickyma**
### What changes were proposed in this pull request?

Fix get_pid_file_name function for the dashboard.

### Why are the changes needed?

This is the follow up pr of  [#1056](https://github.com/apache/incubator-uniffle/pull/1056)

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

By running the script: start-dashboard.sh and stop-dashboard.sh.

### CODE REVIEW

**Comment by rickyma** [comment]
@jerqi Please have a look at this. Thanks.

**Review by jerqi — APPROVED** [review]
LGTM, thanks @rickyma

**Comment by jerqi** [comment]
cc @yl09099

File: bin/utils.sh [hunk 1]
```diff
@@ -59,14 +59,16 @@ function common_shutdown {
...
-# args:               Process name, coordinator or shuffle-server
+# args:               Process name, coordinator, shuffle-server or uniffle-dashboard
 #---
 function get_pid_file_name {
   process_name="$1"
   if [[ $process_name == "coordinator" ]]; then
     pid_file="coordinator.pid"
   elif [[ $process_name == "shuffle-server" ]]; then
     pid_file="shuffle-server.pid"
+  elif [[ $process_name == "uniffle-dashboard" ]]; then
```
Comment (yl09099) [inline bin/utils.sh:70]: It's called dashboard.Too long.

Comment (rickyma) [reply bin/utils.sh]: OK, I've already renamed it to "dashboard".

Comment (yl09099) [reply bin/utils.sh]: LGTM, thanks

**Review by jerqi — APPROVED** [review]

**Comment by jerqi** [comment]
@rickyma @yl09099 thanks, merged to master.

**Pull request merged**